compiler/cpp/CMakeLists.txt

```diff
@@ -146,7 +146,7 @@ if(${WITH_PLUGIN})
         src/thrift/plugin/plugin_output.cc
         src/thrift/plugin/plugin.cc
     )
-    TARGET_INCLUDE_DIRECTORIES_THRIFT(thriftc PUBLIC ${Boost_INCLUDE_DIRS})
+    include(BoostMacros)
```

Comment:
Just including BoostMacros doesn't do anything, was there supposed to be something else?

Reply:
It's better to let the original reporter resolve the issue they identified in GitHub.